chart/templates/cluster-restricted-root-podsecuritypolicy.yaml

```diff
@@ -39,7 +39,8 @@ spec:
     - 'secret'
     - 'emptyDir'
     - 'persistentVolumeClaim'
-  hostNetwork: false
+    - 'hostPath'
```

Comment:
`hostNetwork` is rather mighty - I'd vote for introducing a separate PSP for registry-facade.